cmd/ps.go

```diff
@@ -125,3 +146,35 @@ func printShipmentStatus(name string, shipment ComposeShipment) {
 
 	fmt.Println("-----")
 }
+
+//returns the primary port of a shipment
+func getShipmentPrimaryPort(dockerCompose project.APIProject, shipment ComposeShipment) (string, error) {
+
+	//get primary port of 1st service
+	for _, container := range shipment.Containers {
+		serviceConfig, success := dockerCompose.GetServiceConfig(container)
+		if !success {
+			log.Fatal("error getting service config")
+		}
+
+		parsedPort := strings.Split(serviceConfig.Ports[0], ":")
+		return parsedPort[0], nil
+	}
+	return "", errors.New("no shipment port found")
+}
+
+//returns a shipment endpoint
+func getShipmentEndpoint(shipment string, environment string, provider string, port string) string {
+	//80 -> http://xxx
+	//443 -> https://xxx
+	//? -> http://xxx:?
+	protocol := "http"
```

Comment:
Currently the `ps` command only works off of yaml files and the helm api which doesn't have that info.